geometry_msgs/Pose2D goal
---
bool success
